src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/ErrorPage.css

```diff
@@ -168,3 +152,42 @@ a {
         color: #13709e;
         text-decoration: underline;
     }
+
+.showRawException {
+    font-size: 1.2em;
+    cursor: pointer;
```

Comment:
Ah ok. For some reason I did not see the tab stop happen and so thought of changing it. Will investigate